src/jvmTest/kotlin/com/charleskorn/kaml/JvmYamlReadingTest.kt

```diff
@@ -42,5 +44,18 @@ class JvmYamlReadingTest : DescribeSpec({
                 result shouldBe 123
             }
         }
+
+        describe("reading an empty yaml without throwing an error") {
```

Comment:
These changes aren't specific to the JVM, but this file is for tests that only apply to the JVM. Could you please move them to the `commonTest` source set?

Reply:
I tried it but the problem is that the common Kotlin version of YAML only has the default variable. If there were an expected function decodeFromStream or so it would work..

Reply:
The common Kotlin version has a `decodeFromString` method that could be used in the tests.